docs/source/tutorials/stable_diffusion.mdx

```diff
@@ -378,20 +400,43 @@ save_directory = "lcm_sdxl_neuron/"
 stable_diffusion.save_pretrained(save_directory)
 
 # Push to hub
-stable_diffusion.push_to_hub(save_directory, repository_id="my-neuron-repo", use_auth_token=True)
+stable_diffusion.push_to_hub(save_directory, repository_id="Jingya/lcm-sdxl-neuronx", use_auth_token=True)
 ```
 
 ### Text-to-Image
 
 Now we can generate images from text prompts on Inf2 using the pre-compiled model:
 
+***LCM of Stable Diffusion***
+
+```python
+from optimum.neuron import NeuronLatentConsistencyModelPipeline
+
+pipe = NeuronLatentConsistencyModelPipeline.from_pretrained("Jingya/LCM_Dreamshaper_v7_neuronx")
+
+num_images_per_prompt = 2
+prompt = ["Self-portrait oil painting, a beautiful cyborg with golden hair, 8k"] * num_images_per_prompt
```

Comment:
It should actually be prompts there, no ? It is a bit confuzing to have a batch size of 1 but passing multiple prompts.

Reply:
In the modeling, `prompt` can either be a str or a list of prompts, so I find both work for me

https://github.com/huggingface/optimum-neuron/blob/ac90dc3828e13623b878f19b559998d011f7d66b/optimum/neuron/pipelines/diffusers/pipeline_stable_diffusion_xl.py#L40

Reply:
Sorry, my comment was not specific enough. It is ok to pass multiple prompts, but the normal behaviour for a pipe is to process them one at a time, since you did not specify a batch_size of 1 when instantiating it, and the model has been compiled with batch_size 1 too.
I am assuming here that you will actually process two images in parallel, which should correspond to an actual batch size of 2.
But maybe I am getting it wrong.

Reply:
At least that's the behaviour of the transformers pipelines ... but maybe for diffusers, it is different.

Reply:
LCM currently can only be compiled with bs=1, during the inference, since we are doing dp for the whole pipe, yes exactly we would be able to run bs=2 in parallel.

I know it's confusing that this is not processed inside the pipeline with an argument `num_images_per_prompt`, but we need to pass the prompt twice into the pipeline... Well I would say it's temporary, I hope that Annapurna could unblock me with the compilation issue so that I can better organise the bs along with arguments like `num_images_per_prompt`.